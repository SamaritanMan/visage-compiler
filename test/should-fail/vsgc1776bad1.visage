/**
 * Should-fail test VSGC-1776 : Check for malformed 'run()' function parameters
 *
 * @test/compile-error
 */

function run(arg : String) {
}
