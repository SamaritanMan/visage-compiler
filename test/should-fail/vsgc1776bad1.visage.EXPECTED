test\should-fail\vsgc1776bad1.visage:7: 'run' function must either have no parameters or one parameter of type 'String[]'.
Use:     function run()    or
Or use:  function run(args : String[])
function run(arg : String) {
^
1 error
